.github/workflows/update-schemas.yaml

```diff
@@ -0,0 +1,62 @@
+name: Update schemas in repository
+on:  # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '0 3 * * *'
+
+jobs:
+  update-store:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.INFRA_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets.INFRA_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.SMAUG_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets.SMAUG_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.MORTY_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets. MORTY_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.RICK_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets.RICK_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.BALROG_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets.BALROG_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Prepare commit message
+        id: prepare-commit-message
+        run: |
+          echo ::set-output name=commit-message::"feat: Update schemas - $(date +%Y-%m-%d)"
```

Comment:
No need for this, commit already has an author date embedded in it. Also the commit message should not be `feat:`. This should be `chore: Generate schemas` https://www.conventionalcommits.org/en/v1.0.0/. Static commit message is fine in this case.

Reply:
Resolved in latest commit